### PR TITLE
Siquery printers

### PR DESCRIPTION
**Pull request by SalahKmt**

### CODE REVIEW

File: siquery_tables/src/windows/mod.rs [hunk 1]
```diff
@@ -196,6 +210,11 @@ mod tests {
         fn get_services_file(&self) -> Option<String> {
             Some(String::from(include_str!("../../test_data/services.txt")))
         }
+
+        fn get_wmi_printers_info(&self) -> Option<String>{
+            Some(String::from(include_str!("../../test_data/printers.txt")))
```
Comment (ekse) [inline siquery_tables/src/windows/mod.rs]: printers.txt is missing, also please rename it to wmi-printers.txt.

**Comment by SalahKmt** [comment]
+++ changed the uptime table to return a result instead of a string

File: siquery_tables/test_data/wmi-printers.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Local=TRUE
+Name=Snagit 2018
+PortName=C:\ProgramData\TechSmith\Snagit18\PrinterPortFile
+PrinterStatus=3
+PrintJobDataType=RAW
+PrintProcessor=winprint
+Priority=1
+Status=Unknown
+SystemCreationClassName=Win32_ComputerSystem
+SystemName=DEVOLUTIONS124
```
Comment (awakecoding) [inline siquery_tables/test_data/wmi-printers.txt]: Please edit the computer name to something more generic, like COMPUTER, and edit the corresponding test.

File: siquery_tables/src/windows/mod.rs [hunk 3]
```diff
@@ -105,6 +107,15 @@ impl SystemReaderInterface for SystemReader {
         File::open(file_location + "\\system32\\drivers\\etc\\services").ok()?.read_to_string(&mut string).ok()?;
         Some(string)
     }
+
+    fn get_wmi_printers_info(&self)-> Option<String>{
+        let output = Command::new("wmic")
+            .args(&["printer",
+                "get",
+                "Attributes,Caption,CreationClassName,DeviceID,DoCompleteFirst,DriverNAme,ExtendedPrinterStatus,HorizontalResolution,Local,Name,PortName,PrinterStatus,PrintJobDataType,PrintProcessor,Priority,Status,SystemCreationClassName,SystemName,VerticalResolution",
```
Comment (awakecoding) [inline siquery_tables/src/windows/mod.rs]: minor typo: DriverNAme -> DriverName

**Pull request merged**